src/com/xilinx/rapidwright/design/DesignTools.java

```diff
@@ -3299,13 +3295,24 @@ public static void createCeSrRstPinsToVCC(Design design) {
                     } else {
                         pinName = "RSTREGBU";
                     }
-                    if (si.getSitePinInst(pinName) == null) {
-                        vcc.createPin(pinName, si);
-                    }
+                    maybeCreateVccPinAndPossibleInversion(si, pinName, vcc, gndInvertibleToVcc);
                 }
             }
         }
     }
+    
+    private static void maybeCreateVccPinAndPossibleInversion(SiteInst si, String sitePinName, Net vcc, Net gndInvertibleToVcc) {
```

Comment:
Is this still the most descriptive method name?

Reply:
My slight irk was that this method doesn't do any inversion by itself anymore, since it doesn't set any site PIPs nor do any site routing; rather, it erases some site routing to allow inversion to occur. Let's leave it then.